src/libs/actions/connections/QuickBooksOnline.ts

```diff
@@ -0,0 +1,11 @@
+import {getCommandURL} from '@libs/ApiUtils';
+
+function getQuickBooksOnlineSetupLink(policyID: string) {
+    const params = new URLSearchParams({policyID});
+    const commandURL = getCommandURL({command: 'ConnectPolicyToQuickbooksOnline'});
```

Comment:
This can probably go in here: https://github.com/Expensify/App/blob/fa2bfb29dbcdc84c369c77cfcadda0a8a7a577e7/src/libs/API/types.ts#L8-L8

This method looks pretty generic. Consider maybe something like `getCommandUrlWithParams(command, params)`. Seems like we will have more of these WebView in the future.

Reply:
I'm a bit hesitant because `command: Request` already contain a `data: Record<string, unknown>`, see https://github.com/Expensify/App/blob/67ee4530548d7cdbdb75ff6b1508fd902496bd1b/src/types/onyx/Request.ts#L13-L25

Should we change `getCommandURL` so appends to the URL what we have in `command.data` instead of adding a new `params` argument? or that would be confusing because `data` is supposed to always be in the body?